Tropos/Controllers/SettingsController.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+@objc(TRSettingsController) final class SettingsController: NSObject {
+    private let locale: NSLocale
+
+    var unitSystem: UnitSystem {
+        get {
+            let rawUnitSystem = NSUserDefaults.standardUserDefaults().integerForKey(TRSettingsUnitSystemKey)
+            return UnitSystem(rawValue: rawUnitSystem)!
+        }
+        set {
+            NSUserDefaults.standardUserDefaults().setInteger(newValue.rawValue, forKey: TRSettingsUnitSystemKey)
+        }
+    }
+
+    init(locale: NSLocale) {
+        self.locale = locale
+    }
+
+    convenience override init() {
```

Comment:
Have you considered using `init(locale: NSLocale = .autoupdatingCurrentLocale())` instead of a convenience initializer?